app/scripts/main.js

```diff
@@ -1,13 +0,0 @@
-// /* eslint-disable camelcase */
-// // NHS.UK frontend
-// import nhsuk_header from '../../node_modules/nhsuk-frontend/packages/components/header/header';
-// import nhsuk_skipLink from '../../node_modules/nhsuk-frontend/packages/components/skip-link/skip-link';
-// import autocomplete from '../../node_modules/nhsuk-frontend/packages/components/header/headerAutoComplete';
-
-// // HTML5 polyfills
-// import '../../node_modules/nhsuk-frontend/packages/components/details/details';
-
-// // Initialise components
-// nhsuk_header();
-// nhsuk_skipLink();
-// autocomplete();
```

Comment:
Can we delete this file?

Reply:
I don't know ... can we? I didn't want to delete it then somewhere complain about it not being found. I'll take a look and see

Reply:
🤷‍♂ I can't remember why we commented it out in the first place. But yeah let's see if we can just remove it

Reply:
leaving in as it makes webpack load correctly